src/wp-includes/comment.php

```diff
@@ -4103,3 +4108,30 @@ function _wp_check_for_scheduled_update_comment_type() {
 		wp_schedule_single_event( time() + MINUTE_IN_SECONDS, 'wp_update_comment_type_batch' );
 	}
 }
+
+/**
+ * Register notes metadata for notes status.
+ *
+ * @since 6.9.0
+ */
+function wp_register_notes_metadata() {
```

Comment:
I'd model this closer to `wp_create_initial_post_meta`, so `wp_create_initial_comment_meta`. Future additions would be added into that one function, instead of splitting each registration into it's own function.

Reply:
Fixed in 48cb687fc291eb33e03dd581f1664104558885d1